acrobat/blocks/dc-converter-widget/dc-converter-widget.js

```diff
@@ -113,7 +113,23 @@ const verbRedirMap = {
   'number-pages': 'number',
 };
 
+const exhLimitCookieMap = {
+  'to-pdf': 'cr_p_c_e_ac',
+  'pdf-to': 'ex_p_c_e_ac',
+  'compress-pdf': 'cm_p_ops_e_ac',
+  'rotate-pages': 'or_p_c_e_ac',
+};
+
+// TODO: Retire once app id is unified with chrome ex viewer
+const exhStartLimitCookieMap = {
+  'to-pdf': 'cr_p_c_st_e_ac',
+  'pdf-to': 'ex_p_c_st_e_ac',
+  'compress-pdf': 'cm_p_ops_st_e_ac',
+  'rotate-pages': 'or_p_c_st_e_ac',
+};
```

Comment:
wondering if you changed the naming scheme to put the `_st` at the end, then you could get away with just searching for the common prefix. One map, one search.